wordpress-mobile/WordPress-Android
# Revisions visual preview pt2 (visual preview)

This PR adds a visual preview to revision.

I was considering adding a visual preview to `HistoryDetailFragment`, but with the current implementation, the easiest place was `HistoryDetailContainerFragment`, as we can more easily control the state, menu and don't have to access fragments inside `ViewPager` adapter.

PS: I also tried to extract Aztec specific XML attributes to a separate style without much luck :(

To test:

1. Open a revision and using overflow menu in the toolbar switch to visual preview.
2. Make sure visual preview works as expected.

Labels when switching to visual preview:
[![Image from Gyazo](https://i.gyazo.com/8ecdee24f8492256f79f8d483463b568.png)](https://gyazo.com/8ecdee24f8492256f79f8d483463b568)
Label when switching back to HTML preview:
[![Image from Gyazo](https://i.gyazo.com/2ef67dcb5722f1b9ac6459c046c0516a.png)](https://gyazo.com/2ef67dcb5722f1b9ac6459c046c0516a)

Visual preview with a title:
[![Image from Gyazo](https://i.gyazo.com/94a8e7bea4c0775fa0d740cacf3fda79.png)](https://gyazo.com/94a8e7bea4c0775fa0d740cacf3fda79)

Visual preview without a title:
[![Image from Gyazo](https://i.gyazo.com/29c818015306a58cb49f72446b996b93.png)](https://gyazo.com/29c818015306a58cb49f72446b996b93)
